src/modules/emote_menu/stores/emote-menu-view-store.js

```diff
@@ -40,6 +47,19 @@ function createCategory(id, provider, displayName, icon, categoryEmotes = []) {
   };
 }
 
+let categoryOrder = storage.get(EmoteCategoriesOrderStorageKey);
+
+function organizeCategories(categories) {
+  if (categoryOrder == null) {
+    categoryOrder = categories.map(({category}) => category.id);
+    storage.set(EmoteCategoriesOrderStorageKey, categoryOrder);
```

Comment:
we probably do not need to persist a default value here. instead, we should only persist changes that deviate from default behavior